### PR TITLE
Add more information about GitHub Teacher

### PR DESCRIPTION
**Pull request by crichID**
This pull request is in response to #3 and addresses the following action item:

- Links to the On Demand community

It also removes some of the template blog posts so we can begin adding our own!

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -2,14 +2,14 @@
 title: GitHub Teacher's Awesome Portfolio
 header-img: img/home-bg.jpg
 email: services@github.com
-copyright_name: Your/Project/Corporate Name
+copyright_name: GitHub, Inc.
```
Comment (githubteacher) [inline _config.yml:5]: Thanks for this change - we hadn't caught this yet!

File: _config.yml [hunk 2]
```diff
@@ -2,14 +2,14 @@
...
 description: "A portfolio for githubteacher"
 baseurl:
 url: "https://githubteacher.github.io"
 twitter_username: githubtraining
 github_username:  githubteacher
 facebook_username:  github
-email_username:  your-email@yourdomain.com
+email_username:  services@github.com
```
Comment (githubteacher) [inline _config.yml:12]: 👍 This is much better!

File: about.html [hunk 3]
```diff
@@ -1,14 +1,13 @@
...
+<p>Thank you for visiting my blog. I am the teacher of all things GitHub and this blog is a place I can share my random learnings and connect with you, the GitHub community.</p>
 
-<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Eius praesentium recusandae illo eaque architecto error, repellendus iusto reprehenderit, doloribus, minus sunt. Numquam at quae voluptatum in officia voluptas voluptatibus, minus!</p>
-
-<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Nostrum molestiae debitis nobis, quod sapiente qui voluptatum, placeat magni repudiandae accusantium fugit quas labore non rerum possimus, corrupti enim modi! Et.</p>
-
-<a href="htp:/broken.z"> broken link </a>
+<p>If you are interested in learning how to use Git and GitHub, check out these resources:</p>
+<ul>
+  <li><a href="htt://services.github.com/on-demand">GitHub's On-Demand Training Classes</a></li>
```
Comment (githubteacher) [inline about.html]: This link doesn't look correct. Can you please change it to `http://services.github.com/on-demand`?

**Review by githubteacher — CHANGES_REQUESTED** [review]
@crichID Just one small change to a link, all the rest looks good! Thank you!

**Review by brianamarie — APPROVED** [review]
Much better - thank you! 🙇

**Review by githubteacher — APPROVED** [review]

**Pull request merged**